cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml

```diff
@@ -171,4 +171,8 @@ data:
             proxy_pass http://model/isAdminAuthenticated;
         }
     {{- end }}
+        location / {
+            expires max;
```

Comment:
I know we didn't change the ETAG but doesnt this mean we never check an ETag bump unless you explicitly tell Chrome, ie hard refresH?

Reply:
I will test. Based on [this](https://web.dev/http-cache/) you are probably right. In that case, what should we do? Make the cache 24 hours? Ultimately, cache busting in React will eliminate this issue altogether, but in the meantime we'll have an interesting balance to strike.

Reply:
Niko, what'd you learn in testing?

Reply:
Didn't test today, but it's on the list for the week. Happy to move it up in priority.